client/resourcemanager_client.go

```diff
@@ -248,35 +268,51 @@ func (c *client) createTokenDispatcher() {
 func (c *client) handleResourceTokenDispatcher(dispatcherCtx context.Context, tbc *tokenBatchController) {
 	var connection resourceManagerConnectionContext
 	if err := c.tryResourceManagerConnect(dispatcherCtx, &connection); err != nil {
-		log.Warn("get stream error", zap.Error(err))
+		log.Warn("[resource_manager] get token stream error", zap.Error(err))
 	}
 
+	toReconnect := false
 	for {
+		// Fetch the request from the channel.
 		var firstRequest *tokenRequest
```

Comment:
How to change `var firstRequest *tokenRequest` to `case firstRequest := <-tbc.tokenRequestCh:`,
and move all the code inside the case

Reply:
I refine this part to make sure we can avoid creating variables in the loop. PTAL again.